### PR TITLE
add command to simplify service port forwarding for development

### PR DESCRIPTION
**Pull request by christianvogt**
## Description

With the introduction of proxied services requiring port forwarding for local development, it is cumbersome to setup the commands necessary for local development.

This PR adds a make target to simplify port-forwarding for the various proxied services.

```
NAMSPACE=my-project make port-forward
```

If one of the services does not exist, then port forwarding will fail only for that service.

## How Has This Been Tested?

Locally running command in description.

Output:
```
▶ NAMESPACE=my-project make port-forward
parallel -j0 --lb ::: \
        'oc port-forward -n my-project svc/ds-pipeline-metadata-envoy-dspa 9090:9090' \
        'oc port-forward -n my-project svc/ds-pipeline-dspa 8443:8443' \
        'oc port-forward -n my-project svc/-tls 9443:443'
Error from server (NotFound): services "trustyai-service-tls" not found
Forwarding from 127.0.0.1:9090 -> 9090
Forwarding from [::1]:9090 -> 9090
Forwarding from 127.0.0.1:8443 -> 8443
Forwarding from [::1]:8443 -> 8443
```

Missing namespace:
```
▶ make port-forward    
Makefile:80: *** Missing NAMESPACE variable.  Stop.
```

## Test Impact

N/A

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

cc @alexcreasy @lucferbux @Gkrumbach07

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,7 +52,7 @@ ifdef OC_TOKEN
 	oc login ${OC_URL} --token=${OC_TOKEN}
 else
 	$(info **** Using OC_USER and OC_PASSWORD for login ****)
-	oc login ${OC_URL} -u ${OC_USER} -p ${OC_PASSWORD} --insecure-skip-tls-verify=true
+	oc login ${OC_URL} -u ${OC_USER} -p "${OC_PASSWORD}" --insecure-skip-tls-verify=true
```
Comment (lucferbux) [inline Makefile:55]: is it not working if it's not a string?

Comment (christianvogt) [reply Makefile:55]: I had a password containing a `)` and I get a syntax error

Comment (lucferbux) [reply Makefile:55]: ahhh nice, good catch then!

**Review by lucferbux — APPROVED** [review]
/lgtm

This will helps us too with model registry, I'll adapt it in my PR.

**Pull request merged**